packages/dmn-editor/tests/e2e/__fixtures__/nodes.ts

```diff
@@ -60,6 +60,11 @@ export class Nodes {
     return (await this.get({ name: args.name }).getAttribute("data-nodeid")) ?? "";
   }
 
+  public async delete(args: { name: string }) {
+    await this.page.getByTitle(args.name, { exact: true }).click({ position: { x: 20, y: 20 } });
+    await this.get({ name: args.name }).press("Delete");
+  }
```

Comment:
Two points here:
- I use click {20, 20}, not click {0, 0}, the second one was not working especially for shapes like input data, that basically at coordinates {0,0}, is out of shape, if we draw div rectangle, so I believe, this is acceptable to use {20, 20}
- we press Delete over given node, not over whole diagram as we do for rename, I do not know why, the second one was not working for deleting, maybe it is related how we bind delete handlers in source code?